migrations/1662648153132_create-unique-constraint-for-title-in-tags-table.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable camelcase */
```

Comment:
@beblicarl please what does this do?

Reply:
it does nothing, i should have removed it. I have no camelCase issues

Reply:
okay, kindly do if not being used